src/inmanta/parser/plyInmantaParser.py

```diff
@@ -68,6 +68,7 @@
     ("nonassoc", "NOT"),
     ("left", "IN"),
     ("left", "RELATION_DEF", "TYPEDEF_INNER", "OPERAND_LIST", "EMPTY", "NS_REF", "VAR_REF", "MAP_LOOKUP"),
+    ("nonassoc", "IS_DEFINED"),
```

Comment:
Why is this?

Reply:
Useless indeed since no shift-reduce conflict in this implementation